Src/VimWpf/Implementation/Mouse/VimMouseProcessor.cs

```diff
@@ -1,34 +1,55 @@
 ﻿using Microsoft.VisualStudio.Text.Editor;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Windows.Input;
+using System.Windows.Threading;
 
 namespace Vim.UI.Wpf.Implementation.Mouse
 {
     internal sealed class VimMouseProcessor : MouseProcessorBase
     {
+        /// <summary>
+        /// How many lines per second we should scroll when auto-scrolling
+        /// </summary>
+        private static readonly int s_linesPerSecond = 8;

```

Comment:
Is there a reason behind 8 here? Or is it just a number that you found works well with auto-scrolling?

Reply:
Manually tuned. I started at 20, dropped it to 10 and then dropped it to 5, which was too slow. There is a scroll multiplier the further away from the edge of the viewport the mouse is, so the user can get 8 / 16 / 24 / 32 ... etc. line per second scrolling.